packages/core/src/components/pagination/stories/pagination.args.ts

```diff
@@ -0,0 +1,32 @@
+import { Category } from '@atomium/storybook-utils/enums/table'
+
+export const PaginationStoryArgs = {
+  parameters: {
+    docs: {
+      description: {
+        component: '',
```

Comment:
**suggestion:** A pagination component is an interface that allows users to navigate through large amounts of content that has been split into multiple pages

Maybe this description would be good for pagination? 🤔

Reply:
nice! done in [0cdc3d5](https://github.com/juntossomosmais/atomium/pull/647/commits/0cdc3d51ba4236f7297b37c5ef60d9632aeefaae)